addons/storyshots/stories/required_with_context/Button.stories.js

```diff
@@ -3,7 +3,7 @@ import React from 'react';
 import { storiesOf } from '@storybook/react'; // eslint-disable-line
 import { action } from '@storybook/addon-actions'; // eslint-disable-line
 
-import Button from './Button';
+import Button from '@storybook/components/dist/demo/Button';
```

Comment:
Can we export `Button` directory from the package? Importing from a sub-path doesn't seem kosher to me.